discovery/manager_test.go

```diff
@@ -25,11 +25,13 @@ import (
 	"time"
 
 	"github.com/go-kit/kit/log"
-	"github.com/prometheus/client_golang/prometheus"
-	dto "github.com/prometheus/client_model/go"
+	client_testutil "github.com/prometheus/client_golang/prometheus/testutil"
```

Comment:
Is there really a need for a named import? `testutil` is not used as a name in this file, and I think it fits nicely in here.